vocabularies/voc4cat/0000255.ttl

```diff
@@ -0,0 +1,13 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0000255>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0000255"^^xsd:token ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0000181> ;
+    skos:definition "The ability of a material to interact with an externally applied magnetic field."@en ;
```

Comment:
The term "ability" does not match so well with "property".
What about: "The characteristics of a material or substance that determine how it interacts or responds to a magnetic field."